GetServerHardwareList only returns 32 Servers or panics

### Scenario/Intent
ovc.GetServerHardwareList(filters, "", "", "", "") only returns 32 Servers

### Environment Details
 - **Golang binding for HPE OneView Version:** 1.80


### Steps to Reproduce
Use the serverhardware.go example from the repo against an appliance with more then 32 servers

### Expected Result
getting all servers like it will be done with Powershell or Python.
curl itself also does only get the first 32 entries.
Please refer to count value and nextpageURI
HPE internally discussed here: https://web.yammer.com/main/threads/eyJfdHlwZSI6IlRocmVhZCIsImlkIjoiMTExMjk1NTY5NjYxOTUyMCJ9


### Actual Result

panic: runtime error: index out of range [32] with length 32

goroutine 1 [running]:
github.com/HewlettPackard/oneview-golang/ov.(*OVClient).GetServerHardwareList(0xc0000cc000, 0xc0000134c0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/skoch/go/src/github.com/HewlettPackard/oneview-golang/ov/server_hardware.go:302 +0xac5
main.main()
        /home/skoch/serverhw.go:25 +0x23b
exit status 2

wit this option i am getting the panic also

....
serverList : CTC H5 HE01, bay 9
serverList : CTC H5 HE11, bay 5
panic: runtime error: index out of range [32] with length 32

goroutine 1 [running]:
github.com/HewlettPackard/oneview-golang/ov.(*OVClient).GetServerHardwareList(0xc0000ca000, 0xc0001382d0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, 0x71e1b1, 0x2, ...)
        /home/skoch/go/src/github.com/HewlettPackard/oneview-golang/ov/server_hardware.go:302 +0xac5
main.main()
        /home/skoch/serverhw.go:25 +0x24b
exit status 2

--- a/examples/server_hardware.go
+++ b/examples/server_hardware.go
@@ -2,9 +2,10 @@ package main
 
 import (
 	"fmt"
-	"github.com/HewlettPackard/oneview-golang/ov"
 	"os"
 	"strconv"
+
+	"github.com/HewlettPackard/oneview-golang/ov"
 )
 
 func main() {

--- a/ov/server_hardware.go
+++ b/ov/server_hardware.go
@@ -21,10 +21,12 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"strconv"
+	"strings"
+
 	"github.com/HewlettPackard/oneview-golang/rest"
 	"github.com/HewlettPackard/oneview-golang/utils"
 	"github.com/docker/machine/libmachine/log"
-	"strings"
 )
 
 // HardwareState
@@ -293,7 +295,12 @@ func (c *OVClient) GetServerHardwareList(filters []string, sort string, start st
 		return serverlist, err
 	}
 
-	for i := 0; i < serverlist.Total; i++ {
+	if count == "" {
+		total := strconv.Itoa(serverlist.Total)
+		return c.GetServerHardwareList(filters, "", "", total, "")
+	}
+
+	for i := 0; i < serverlist.Count; i++ {
 		serverlist.Members[i].Client = c
 	}
 

